fluent-temporal/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "@fluent/temporal",
+  "description": "Experimental Temporal support for Project Fluent",
+  "version": "0.1.0",
+  "homepage": "https://projectfluent.org",
+  "author": "Mozilla <l10n-drivers@mozilla.org>",
+  "license": "Apache-2.0",
+  "contributors": [
+    {
+      "name": "Konstantin Haase",
```

Comment:
Put myself in here. You can take me out, though. I'm happy to "gift" this little library to Mozilla.